### PR TITLE
Monitor Enhancements release notes

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create release notes for the Monitor updates.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/SUMO-240929

### CODE REVIEW

File: blog-service/2024-04-16-apps.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - apps
+  - monitors
+  - terraform
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce an update of all our terraform based monitors. Below are the list of enhancements that were a part of this update.
```
Comment (jpipkin1) [inline blog-service/2024-04-16-apps.md:14]:
```suggestion
We're excited to announce an update of all our Terraform based monitors. Below are the list of enhancements that were a part of this update.
```

**Review by jpipkin1 — APPROVED** [review]
@JV0812, approved contingent on addressing my comments. You'll also have to change the date of the release note.

File: blog-service/2024-04-16-apps.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Monitor Enhancements (Apps)
```
Comment (kimsauce) [inline blog-service/2024-04-16-apps.md:2]:
```suggestion
title: Enhancements to Terraform-Based Monitors (Apps)
```

File: blog-service/2024-04-16-apps.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce an update of all our terraform based monitors. Below are the list of enhancements that were a part of this update.
+
+- Updated Kubernetes monitors to make it compatible with new helm chart versions.
+- Upgraded AWS and Sumologic provider versions for CVE fixes.
```
Comment (kimsauce) [inline blog-service/2024-04-16-apps.md:17]: I have a couple edits on top of John's

```suggestion
- Updated Kubernetes monitors for compatibility with new Helm chart versions.
- Upgraded AWS and Sumo Logic provider versions to address CVE fixes.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
made some edits for clarity

**Pull request merged**